frontend/src/components/app.jsx.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react/addons';
+import {RouteHandler} from 'react-router';
+import {Link} from 'react-router';
+
+let App = React.createClass({
+  render: () => {
+    return (
+      <div className="appContainer">
+        <RouteHandler/>
+        <Link to='sprint'>Sprint</Link>
```

Comment:
OCD: are we supposed to be using double quotes within all HTML tags - you have double quotes in routes.jsx.js